packages/core/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe core SDK package will be documented in this file.
 
+## Recent Changes
+
+- Fixed inconsistencies in punctuation for command descriptions by adding missing periods. [66](https://github.com/zowe/zowe-cli/issues/66)
```

Comment:
No need to update changelogs for packages where nothing changed. Based on the PR diff, it looks like only files inside the folder "packages/cli" have changed.

Please undo the changelog updates for all the other packages that didn't change: core, provisioning, workflows, zosconsole, zosfiles, zosjobs, zosmf, zostso, zosuss.

If the changelog check in the pipeline fails after you remove these changelogs, there's probably a bug in our change detection logic that we'll need to fix 🙂

Reply:
Alright, yes I believe the change detection logic was failing before and was telling me to update the changelogs for all those packages.

Reply:
Thanks, looks like changelog check is passing now 🙂 When it was failing previously, perhaps there were conflicts with the master branch? I think the changelog check may fail until those conflicts are resolved.